client/main.lua

```diff
@@ -84,8 +72,59 @@ RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
 	end)
 
 	PlayerJob = QBCore.Functions.GetPlayerData().job
+	RequestModel("s_m_m_armoured_01")
```

Comment:
Why are you spawning the peds on player login and resource restart even if user is not using target? There is an export that you can use for this

Reply:
Using ped as indication instead of DrawMarker when Target is false. And using ped as target entity when Target is true

Reply:
Ok